judo-ui-react/src/main/resources/actor/src/components/widgets/AggregationInput.tsx.hbs

```diff
@@ -167,7 +154,7 @@ export const AggregationInput = ({
               }}
               InputProps={ {
                 ...params.InputProps,
-                readOnly: readOnly || !onSet,
+                readOnly: readOnly || !aggregationInputButtonProps.find(a => a.name === 'set')?.onClick,
```

Comment:
`aggregationInputButtonProps.find(a => a.name === 'set')` should be extracted to a `useMemo` block above the return statement, so that we can use these as variables, we can improve performance because these parameters rarely change.

Reply:
[ec0c04e](https://github.com/BlackBeltTechnology/judo-ui-react-template/pull/378/commits/ec0c04ef0c9e729799ce08472123e586dede906b)